handler/router.go

```diff
@@ -18,25 +20,20 @@ func NewHandler(services *service.Service, logger logging.Logger) *Handler {
 
 func (h *Handler) InitRoutes() *gin.Engine {
 	router := gin.New()
+	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
-	router.Use(
-		CorsMiddleware,
-	)
-
-	auth := router.Group("/auth")
+	role := router.Group("/roles")
 	{
-		auth.POST("/sign-in", h.signIn)
+		role.POST("/", h.createRole)
+		role.POST("/roleToPerms", h.bindRoleWithPerms)
```

Comment:
Try to avoid namings like this as much as possible, they aren't rest-like. Your REST resources can contain kind of sub-resources. So the route can be `POST roles/123/permissions`.

Reply:
Or the second option can be just having `POST permissions` and put role id as one of the field in CreatePermission object